timev.rb

```diff
@@ -274,24 +284,55 @@ def self.at(time, subsec = false, unit = :microsecond, in: nil)
     end
   end
 
-  # Returns a new \Time object based on the given arguments.
+  # Returns a new \Time object based on the given arguments,
+  # by default in the local timezone.
   #
   # With no positional arguments, returns the value of Time.now:
   #
-  #   Time.new                                       # => 2021-04-24 17:27:46.0512465 -0500
+  #   Time.new # => 2021-04-24 17:27:46.0512465 -0500
+  #
+  # With one to six arguments, returns a new \Time object
+  # based on the given arguments, in the local timezone.
+  #
+  #   Time.new(2000, 1, 2, 3, 4, 5) # => 2000-01-02 03:04:05 -0600
+  #
+  # There are no minimum and maximum values for +year+;
+  # otherwise, these are the minimum and maximum values:
+  #
+  #   Time.new(-2000, 1, 1, 0, 0, 0)
+  #   # => -2000-01-01 00:00:00 -0600
+  #   Time.new(2000, 12, 31, 23, 59, 60)
+  #   # => 2001-01-01 00:00:00 -0600 # Note the rollover cause by 60 sec.
```

Comment:
We should have the similar description on minimum and maximum values from `Time.utc` here.

Reply:
Added.